pkgs/misc/rivalcfg/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, python3Packages }:
+
+with python3Packages; buildPythonPackage rec {
+  pname = "rivalcfg";
+  version = "3.6.0";
+
+  meta = with stdenv.lib; {
+    description = "Small CLI utility program that allows you to configure SteelSeries Rival gaming mice";
+    license     = licenses.wtfpl;
```

Comment:
you should include a homepage (even if this is the same as the github url) and a maintainer

Reply:
Is the maintainer me? Should I add myself to the maintainers file?

Reply:
If you are happy to be the maintainer you should make a seperate commit adding yourself to the maintainers file

Reply:
maintainer is someone with some working knowledge of a package, and some amount of care